apps/web-giddh/src/app/purchase/purchase-order/purchase-order.component.ts

```diff
@@ -107,6 +126,22 @@ export class PurchaseOrderComponent implements OnInit, OnDestroy {
                 this.purchaseOrderUniqueName = '';
             }
         });
+
+        this.purchaseOrderListFilters$.pipe(takeUntil(this.destroyed$)).subscribe(filters => {
```

Comment:
Please move these subscriptions to ngOnInit method to avoid component instantiation